src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1031,6 +1030,7 @@ std::pair< bool, nuraft::cb_func::ReturnCode > RaftReplDev::handle_raft_event(nu
             auto reqs = sisl::VectorPool< repl_req_ptr_t >::alloc();
             for (auto& entry : entries) {
                 if (entry->get_val_type() != nuraft::log_val_type::app_log) { continue; }
+                if (entry->get_buf_ptr()->size() == 0) { continue; }
```

Comment:
What's the case where we have to handle zero sized buffers now?

Reply:
In raft repl test case, we can also send data size as 0.